client/scripts/views/modals/manage_community_modal/chain_metadata_management_table.ts

```diff
@@ -113,12 +115,12 @@ const ChainMetadataManagementTable: m.Component<IChainOrCommMetadataManagementAt
           placeholder: 'gov.edgewa.re',
           onChangeHandler: (v) => { vnode.state.customDomain = v; },
         }),
-        m(InputPropertyRow, {
+        (app.chain as Token).isToken ? m(InputPropertyRow, {
```

Comment:
There are ethereum communities that are non-token (like AAVE and COMP-based communities) that might *also* have snapshot enabled for off-chain signaling. We should check for `app.chain?.meta.chain.base === 'ethereum'` (or something close to that) instead.